rails/app/views/external_activities/edit_collections.html.haml

```diff
@@ -0,0 +1,54 @@
+.search_materials
+  - material = @material[0]
+  - external_activity_name = material.name
+  - existing_collections = MaterialsCollection.joins(:materials_collection_items).where(:materials_collection_items => {:material_id=>material.id,:material_type=>"ExternalActivity"})
+
```

Comment:
Now that is in the external_activities controller, we can assume there is only 1 material.
And the controller will automatically create the @external_activity variable which can be used instead of `material`